### PR TITLE
Bug fixes for libvirt/QEMU orchestrator.

### PR DESCRIPTION
**Pull request by cwize1**
1. Don't use `dmesg` if node is local machine, as some OSes (e.g. Fedora 37) will block its usage.

2. Fix OVMF firmware config search filtering to handle new versions of the OVMF firmware files.

### CODE REVIEW

File: lisa/operating_system.py [hunk 1]
```diff
@@ -383,15 +383,20 @@ def capture_system_information(self, saved_path: Path) -> None:
...
-        find_tool = self._node.tools[Find]
-        file_list = find_tool.find_files(
-            self._node.get_pure_path("/var/log/azure/"),
-            file_type="f",
-            sudo=True,
-            ignore_not_exist=True,
-        )
-        if len(file_list) > 0:
-            self._node.tools[Chmod].update_folder("/var/log/azure/", "a+rwX", sudo=True)
+        file_list = []
```
Comment (squirrelsc) [inline lisa/operating_system.py]: @LiliDeng is working on this, and mostly done.

https://github.com/microsoft/lisa/pull/2517

File: lisa/sut_orchestrator/libvirt/platform.py [hunk 2]
```diff
@@ -195,10 +195,11 @@ def _cleanup(self) -> None:
         if self.platform_runbook.capture_libvirt_debug_logs:
             self._disable_libvirt_debug_log()
 
-        dmesg_output = self.host_node.tools[Dmesg].get_output(force_run=True)
-        dmesg_path = self.host_node.local_log_path / "dmesg.txt"
-        with open(str(dmesg_path), "w") as f:
-            f.write(dmesg_output)
+        if self.host_node.is_remote:
```
Comment (squirrelsc) [inline lisa/sut_orchestrator/libvirt/platform.py:198]: With Lili's change, we introduced a new flat called "is_test_target". Please consider if it's applicable here.

Comment (cwize1) [reply lisa/sut_orchestrator/libvirt/platform.py:198]: `host_node` will never be the test target. So, I think `is_remote` is correct here.

**Review by anirudhrb — DISMISSED** [review]

File: lisa/operating_system.py (hunk 1 above)
Comment (squirrelsc) [reply lisa/operating_system.py]: #2517 was merged. please rebase to the latest code.

**Comment by cwize1** [comment]
@squirrelsc Friendly ping.

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**